spec.bs

```diff
@@ -144,6 +144,10 @@ spec: RFC8941; urlPrefix: https://www.rfc-editor.org/rfc/rfc8941.html
     text: structured header; url: #section-1
     for: structured header
       text: token; url: name-tokens
+spec: CSS; urlPrefix: https://www.w3.org/TR/cssom-view-1/
+  type: dfn
+    text: scroll a target into view; url: #scroll-an-element-into-view
```

Comment:
Per https://dontcallmedom.github.io/webdex/s.html#scroll%20a%20target%20into%20view%40%40cssom-view%25%25dfn it seems this is exported?